db/document.go

```diff
@@ -65,6 +65,7 @@ type SyncData struct {
 	Expiry          *time.Time          `json:"exp,omitempty"`           // Document expiry.  Information only - actual expiry/delete handling is done by bucket storage.  Needs to be pointer for omitempty to work (see https://github.com/golang/go/issues/4357)
 	Cas             string              `json:"cas"`                     // String representation of a cas value, populated via macro expansion
 	Crc32c          string              `json:"value_crc32c"`            // String representation of crc32c hash of doc body, populated via macro expansion
+	Crc32UserXattr  string              `json:"user_xattr_value_crc32"`  // String representation of crc32c hash of user xattr
```

Comment:
```suggestion
	Crc32cUserXattr  string              `json:"user_xattr_value_crc32c"` // String representation of crc32c hash of user xattr
```

Reply:
Agree but leaving this for now as we may change this to something like 'meta_crc32c' so this could be expanded in the future.

Reply:
If we're leaving this, even as a placeholder, I think it makes sense to fix the crc32c in the name.